### ISSUE TITLE
Weird language selector scrolling

### ISSUE THREAD

**Issue by lzk228** [space-wizards/SS14.Launcher#157]
- its going over the edges of the window
- scrolling available only when mouse hovers selection window

https://github.com/space-wizards/SS14.Launcher/assets/124214523/87b53750-38c4-4c99-94ab-5f583cd896b9

**Comment by Visne** [comment]
Is this still an issue for you on latest master? For me it still goes out of the window (not sure if that's really an issue) but everything else seems to work as expected.

**Comment by lzk228** [comment]
> Is this still an issue for you on latest master? For me it still goes out of the window (not sure if that's really an issue) but everything else seems to work as expected.

yes, it is still as on the video, goes out of the window and scrolling available only in the language box

**Comment by PJB3005** [comment]
Still happens for me on master as I just had it happen while testing some language stuff.
